src/main/java/nl/novi/techiteasy/controllers/TelevisionController.java

```diff
@@ -55,4 +57,22 @@ public ResponseEntity<Void> deleteTelevision(@PathVariable Long id) {
         return ResponseEntity.noContent().build();
     }
 
+    @PutMapping("/{id}/remotecontroller")
```

Comment:
Je mist in je endpoint de id van de remotecontroller. Dus /{id}/remotecontroller/{remoteid} (of iets dergelijks).
Misschien wordt hier je loop door veroorzaakt?